src/extension/mapml/src/main/java/org/geoserver/mapml/MapMLDocumentBuilder.java

```diff
@@ -2706,4 +2762,38 @@ public static boolean isWMSOrWMTSStore(LayerInfo layerInfo) {
         }
         return false;
     }
+
+    public static double calculateBestScaleDenominator(
```

Comment:
Does not seem to be in use (in this PR at least)?

Reply:
Removed